Library/Homebrew/sandbox.rb

```diff
@@ -162,6 +162,11 @@ class SandboxProfile
           (regex #"^/dev/ttys?[0-9]*$")
           )
       (deny file-write*) ; deny non-whitelist file write operations
+      (deny file-read*
```

Comment:
Could also use `HOMEBREW_PREFIX/"lib"`, etc in that method rather than hardcoding `/usr/local`, if desired.